src/test/test-unaligned.c

```diff
@@ -159,7 +159,26 @@ static void test_le(void) {
         assert_se(memcmp(&scratch[7], &data[7], sizeof(uint64_t)) == 0);
 }
 
+static void test_ne(void) {
+        uint16_t x = 4711;
+        uint32_t y = 123456;
+        uint64_t z = 9876543210;
```

Comment:
true. but the alignment checks are in place already for the two le/be flavours of the function. this test just checks that the function is mapped correctly to the ne flavour too.